cfme/common/vm.py

```diff
@@ -717,29 +718,15 @@ def equal_drift_results(self, row_text, section, *indexes):
         tb.select("Select up to 10 timestamps for Drift Analysis")
 
         # Make sure the section we need is active/open
-        sec_loc_map = {
-            'Properties': 'Properties',
-            'Security': 'Security',
-            'Configuration': 'Configuration',
-            'My Company Tags': 'Categories'}
-        sec_loc_template = "//div[@id='all_sections_treebox']//li[contains(@id, 'group_{}')]" \
-                           "//span[contains(@class, 'dynatree-checkbox')]"
-        sec_checkbox_loc = "//div[@id='all_sections_treebox']//li[contains(@id, 'group_{}')]" \
-            "//span[contains(@class, 'dynatree-checkbox')]".format(sec_loc_map[section])
         sec_apply_btn = "//div[@id='accordion']/a[contains(normalize-space(text()), 'Apply')]"
 
         # Deselect other sections
-        for other_section in sec_loc_map.keys():
-            other_section_loc = sec_loc_template.format(sec_loc_map[other_section])
-            other_section_classes = sel.get_attribute(other_section_loc + '/..', "class")
-            if other_section != section and 'dynatree-partsel' in other_section_classes:
-                # Element needs to be checked out if it has no dynatree-selected
-                if 'dynatree-selected' not in other_section_classes:
-                    sel.click(other_section_loc)
-                sel.click(other_section_loc)
+        for other_section in drift_section.child_items():
+            drift_section.check_node(other_section.text)
```

Comment:
It is hierarchic tree and some child items selected but not all of them, so it is done to  check all of them